dask/dataframe/core.py

```diff
@@ -2745,6 +2745,41 @@ def _repr_html_(self):
         return self._HTML_FMT.format(data=data, name=key_split(self._name),
                                      task=len(self.dask))
 
+    def _select_columns_or_index(self, columns_or_index):
+        """
+        :param columns_or_index: Column name or list of column names
+        and/or the name of the DataFrame's index
+        :return: Dask DataFrame with columns corresponding to each column or
+        index level in columns_or_index.  If included, the column corresponding 
+        to the index level is named _index
+        """
+        # Ensure columns_or_index is a list
+        columns_or_index = (columns_or_index
+                            if isinstance(columns_or_index, list)
```

Comment:
Maybe rewrite this to check if `columns_or_index` is a sequence, or maybe more easily reverse the logic and check for `isinstance(columns_or_index, pd.compat.string_types)`

Reply:
Working on some updates now.  I did the check this way because strings, integers, and tuples can all be valid column keys for pandas and I'm aiming to end up with a `List` of column keys. Does that make sense?